src/funfuzz/js/jsfunfuzz/gen-grammar.js

```diff
@@ -1398,7 +1664,7 @@ var typedArrayConstructors = [
   "Uint32Array",
   "Float32Array",
   "Float64Array",
-  "Uint8ClampedArray"
+  "Uint8ClampedArray",
```

Comment:
Trailing comma.  I wasn't aware that this was eslint's default behavior.  You can enable checking of this via https://eslint.org/docs/rules/comma-dangle